chat-plugins/trivia.js

```diff
@@ -727,6 +704,12 @@ var commands = {
 	}
 };
 
+var room = Rooms.get('trivia');
```

Comment:
The only issue would be that if a new chatroom called Trivia were created, neither of these would be exported until server restart.